### DISCUSSION TITLE
Add limit for git log

### DISCUSSION THREAD

**Discussion by chaopeng** [PatrickF1/fzf.fish#318] [Ideas]
Thank you so much for building this useful plugin.

`_fzf_search_git_log` is very helpful, but it take a long time for large repo, for example chromium, It would be good if we can have limit when git log query, [-n <number> / --max-count=<number>](https://git-scm.com/docs/git-log#Documentation/git-log.txt--nltnumbergt).

For sure, this is edge case and not needed by everyone.

My idea is to accept an env `_fzf_search_git_log_limit` to set limit.

**Comment by PatrickF1** [comment]
Hi, doesn’t it stream in the results anyway? I don’t understand why it
would take a long time if you can start viewing and searching immediately.

**Comment by chaopeng** [comment]
Oh, I re-try again the view and result start immediately.
